nautobot/virtualization/filters.py

```diff
@@ -233,8 +296,35 @@ class VMInterfaceFilterSet(BaseFilterSet, StatusModelFilterSetMixin, CustomField
     mac_address = MultiValueMACAddressFilter(
         label="MAC address",
     )
-    tag = TagFilter()
+    tagged_vlans_vid = django_filters.ModelMultipleChoiceFilter(
+        field_name="tagged_vlans__vid",
+        queryset=VLAN.objects.all(),
+        to_field_name="vid",
+        label="Tagged VLAN (VID)",
+    )
+    tagged_vlans = django_filters.ModelMultipleChoiceFilter(
+        field_name="tagged_vlans",
+        queryset=VLAN.objects.all(),
+        label="Tagged VLAN",
+    )
```

Comment:
Would it make sense to unify these two into a single `NaturalKeyOrPKMultipleChoiceFilter` using `vid` as the natural key field?